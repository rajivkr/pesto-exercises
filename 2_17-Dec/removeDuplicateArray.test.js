const removeDuplicatesInArray = require('./removeDuplicateArray')
  .removeDuplicatesInArray;

describe('using memoize test cases', () => {
  it('should use it from cache', () => {
    expect(removeDuplicatesInArray([1, 2], [1, 2, 1, 4, 5])).toMatchSnapshot();
    expect(removeDuplicatesInArray([0], [-0])).toMatchSnapshot();
    expect(removeDuplicatesInArray([1, 2], [1, 2, 1, 3, 4])).toMatchSnapshot();
  });
});
