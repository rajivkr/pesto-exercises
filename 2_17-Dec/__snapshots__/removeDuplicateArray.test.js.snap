// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`removing Duplicates test cases should remove from first array 1`] = `
Array [
  4,
  5,
]
`;

exports[`removing Duplicates test cases should remove from first array 2`] = `
Array [
  3,
  4,
]
`;
